scalableminds/webknossos
# Fix groups in multicomponent tree splitter

When uploading an NML with *multiple* trees that each have multiple components, the wrapping tree groups did not get added properly (always using the original passed treeGroups as base again, instead of the previously mutated variable).

### Steps to test:
- upload NML with multiple trees/“things” that (1) already have groups (2) are have multiple components
- should not give an error, instead import those and split the trees as expected

### Issues:
- fixes #4718

------
